Add PiiEntities to Azure.AI.TextAnalytics - closing due to Merge issues

nisha-bhatia opened the pull request:
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

nisha-bhatia committed:
wip

nisha-bhatia committed:
Update TextAnalyticsClient.cs

nisha-bhatia committed:
wip

nisha-bhatia committed:
wip

azure-sdk commented:
**API change check for `Azure.AI.TextAnalytics`**

API changes have been detected in `Azure.AI.TextAnalytics`. You can review API changes [here](https://apiview.dev/Assemblies/Review/4250f17858b54bdaaf92fb86eabbca40?diffOnly=True&diffRevisionId=f08a286d3f5a43ad9f1c16cad078b5af)

**API changes**
```diff
+             V2022_03_01_Preview = 4,
-         public TextAnalyticsClientOptions(ServiceVersion version = V3_2_Preview_2);
+         public TextAnalyticsClientOptions(ServiceVersion version = V2022_03_01_Preview);
+         public static TextAnalyticsOperationStatus PartiallySucceeded { get; }
```

maririos committed:
[TA] Test enabled for Detect Language (#28449)

nisha-bhatia committed:
Add RecognizeEntities to Azure.AI.TextAnalytics (#28433)

jsquire committed:
[Text Analytics] Delegation to Serivce Clients (#28463)

* [Text Analytics] Delegation to Serivce Clients

The focus of these changes is to shift functionality for the Language REST
API integration into the `LanguageServiceClient` and update the `TextAnalyticsClient`
to delegate all operations to the service client.

nisha-bhatia commented on sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClient.cs:
```diff
@@ -1054,34 +1052,30 @@ public virtual async Task<Response<RecognizePiiEntitiesResultCollection>> Recogn
-                var analyzeRecognizeEntities = new AnalyzeTextPiiEntitiesRecognitionInput { AnalysisInput = input };
-                analyzeRecognizeEntities.Parameters.ModelVersion = options.ModelVersion;
-                analyzeRecognizeEntities.Parameters.LoggingOptOut = options.DisableServiceLogs;
+                piiEntitiesRecognitionInput.Parameters = new PiiTaskParameters(options.DisableServiceLogs, options.ModelVersion, new PiiDomain(options.DomainFilter.GetString()), options.CategoriesFilter, Constants.DefaultStringIndexType);
```
https://github.com/Azure/azure-sdk-for-net/issues/28465

nisha-bhatia committed:
Update LanguageServiceClient.cs

nisha-bhatia committed:
merge

nisha-bhatia commented:
closing due to too many file changes
re-opened here: https://github.com/Azure/azure-sdk-for-net/pull/28470
